core/trino-main/src/main/java/io/trino/sql/analyzer/QueryExplainerFactory.java

```diff
@@ -43,7 +43,7 @@ public QueryExplainerFactory(
     {
         this.planOptimizersFactory = requireNonNull(planOptimizersFactory, "planOptimizersFactory is null");
         this.planFragmenter = requireNonNull(planFragmenter, "planFragmenter is null");
-        this.plannerContext = requireNonNull(plannerContext, "metadata is null");
+        this.plannerContext = requireNonNull(plannerContext, "plannerContext is null");
```

Comment:
All the above are irrelevant, but things like that are helpful to reduce the review time. 
As a reviewer, i don't want to spend time on checking messages, and people usually copy them (which is error prone), instead of using templates like https://gist.github.com/findepi/b597d2bc91e5a30c2a44fd65fe252f0b

Reply:
Mismatches like this more often happen because of refactorings: a parameter gets renamed, but the message stays unchanged. Personally, I usually forget to use rnn at all (there could be a check for it too!)